server/game/cards/10-SoD/SpareBoot.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+
+class SpareBoot extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Stand and move attachment',
+            phase: 'challenge',
+            limit: ability.limit.perPhase(1),
+            target: {
+                activePromptTitle: 'Select an attachment',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'attachment' &&
+                                       card.parent && card.parent.controller === this.controller &&
+                                       (card.parent !== this || card.kneeled) &&
+                                       this.controller.canAttach(card, this)
+            },
+            handler: context => {
+                if(context.target.parent !== this) {
+                    context.player.attach(context.player, context.target, this);
```

Comment:
Similarly, the first parameter of the method here is the new controller for the attachment. This will thus take control of the attachment, which isn't correct. Should be `context.player.attach(context.target.controller, ...)` instead.